### PR TITLE
Added transitions for Dark Mode

### LINKED ISSUE #64: Adding transition effects to dark/light theme changes
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
It it would be nice if theme changes are accompanied by transitions into each other. I know this request might sound like too small, but I think it would add little more elegance into the website.

**Additional context**
I would love to add the above changes ! Should I do so ? I am new to GitHub, so should I send a PR first or wait for this issue to get assigned ?

### PR DESCRIPTION
**Pull request by pathakshashank17**
Fixes #64 

- Added transition to Dark/Light theme changes
- Added a 5px margin on some crammed buttons

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -389,6 +389,11 @@ input[type='file']::before {
   grid-auto-columns: 1fr;
 }
 
+.fade-in-dark,
+.fade-in-light {
+  transition: 0.5s linear;
```
Comment (saurabhdaware) [inline css/index.css]: We can make the transition property `0.3s ease`, this way the transition will be faster and fade will look smooth.

**Review by saurabhdaware — CHANGES_REQUESTED** [review]
That's amazing @pathakshashank17! I tested it and it works great! I just mentioned a tiny change below, let me know if that makes sense.

Thanks!

File: css/index.css (hunk 1 above)
Comment (pathakshashank17) [reply css/index.css]: Sure, that looks better ! Will do the changes.

**Review by saurabhdaware — APPROVED** [review]
Perfect! Thank you so much, Shashank! 🌻🥳

**Comment by pathakshashank17** [comment]
Your welcome !

**Pull request merged**